src/dotnet/APIView/APIViewWeb/Client/src/shared/signalr.ts

```diff
@@ -1,10 +1,11 @@
+import { HubConnection } from "@microsoft/signalr";
 import * as hp from "./helpers";
 
 const signalR = require('@microsoft/signalr');
 
-let connection;
-// sender/server side of comment refresh 
-export function PushComment(reviewId, elementId, partialViewResult) {
+let connection: HubConnection;
```

Comment:
You can do signalR.HubConnection. That way you don't have two import(require) statements.